support-frontend/babel.config.json

```diff
@@ -1,14 +1,13 @@
 {
   "presets": [
-    "@babel/preset-react",
+    ["@babel/preset-react", { "runtime": "automatic", "importSource": "@emotion/react" }],
```

Comment:
This was updated for the tests - now we're using `tsc` to transform the `jsx` through the webpack build, but for the tests jest only runs babel, so we need to update the config here too